docs/semgrep-supply-chain/glossary.md

```diff
@@ -30,9 +30,9 @@ A lockfile describes a dependency tree to ensure that deployments and organizati
 
 Semgrep Supply Chain uses lockfiles as part of its analysis to determine the exact version of a dependency that a codebase is using.
 
-## Lockfile-only rules
+## Rules without reachability analysis
```

Comment:
```suggestion
## No-reachability rules
```
Suggestion to at least make it a term here in the glossary.

The hyphen is added between "no" and "reachability" to mean that they are one word. Without the hyphen it becomes "no reachability rules", which is a different meaning entirely.

Reply:
Other ideas:

- "Version-only" or "Version" rules (we literally compare if the library has a vulnerable version)
- "Dependency-only" or "Dependency"

Reply:
Looks like the decision for it has been made, so I'll resolve this